github/search_test.go

```diff
@@ -914,3 +914,42 @@ func TestLabelsSearchResult_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestCommitResult_Marshal(t *testing.T) {
+	testJSONMarshal(t, &CommitResult{}, "{}")
+
+	c := &CommitResult{
+		SHA:         String("test"),
+		HTMLURL:     String("hurl"),
+		CommentsURL: String("curl"),
+		URL:         String("url"),
+		Repository:  &Repository{ID: Int64(1)},
+		Score:       Float64(123),
+		Commit:      &Commit{SHA: String("test")},
+		Author:      &User{ID: Int64(1)},
+		Committer:   &User{ID: Int64(1)},
+		Parents:     []*Commit{},
```

Comment:
Any reason for not supplying a single parent?

Reply:
Never mind, let's get this in before I go OOO.

Reply:
it's just an omitempty so I want to try to ignore this. But Thanks @gmlewis for approving :)